frontend/src/store/playerElement.ts

```diff
@@ -154,6 +180,19 @@ class PlayerElementStore {
         (sub) => sub.srcIndex === playbackManager.currentSubtitleStreamIndex
       );
 
+      const attachedFonts =
+        playbackManager.currentMediaSource?.MediaAttachments?.filter((attach) =>
+          this._isSupportedFonts(attach.MimeType || '')
+        )
+          .map((attach): string | undefined => {
+            if (attach.DeliveryUrl && serverAddress) {
+              return `${serverAddress}${attach.DeliveryUrl}`;
```

Comment:
```suggestion
          .map((a) => {
            if (a.DeliveryUrl && serverAddress) {
              return `${serverAddress}${a.DeliveryUrl}`;
```
To keep it consistent from above.
I also think the type annotations for the return are unnecessary in this case, but ignore that change if locally you get linter complaints.